medic/medic-run.js

```diff
@@ -291,6 +293,29 @@ function failedBecauseNoDevice(output) {
     return NO_DEVICE_PATTERN.test(output);
 }
 
+function onConnect(couchdbURI, retryCount, callback) {
```

Comment:
Please give a more descriptive name to the function, since it's checking if a server is up, not really opening a connection or handling an event (`onX` is a common naming pattern for event handlers).